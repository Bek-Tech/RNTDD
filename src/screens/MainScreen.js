import React, {useState} from 'react';
import RestaurantList from '../components/RestaurantList';
import InputField from '../components/InputField';
import {View, StyleSheet, ScrollView,Dimensions} from 'react-native';

const wHeight = Dimensions.get('window').height

const MainScreen = () => {
  const [restaurantList, setRestaurantList] = useState([]);

  const addRestaurant = (restaurantName, dishName) => {
    const restaurantObj = {restaurantName, dishName, id: Math.random()};
    setRestaurantList([...restaurantList, restaurantObj]);
  };
  return (
      <View style={styles.container}>
        <InputField addRestaurant={addRestaurant} />
        <RestaurantList test="test" restaurantsArr={restaurantList} />
      </View>
  );
};

export default MainScreen;

const styles = StyleSheet.create({
  container:{
    height: wHeight
  }
});
