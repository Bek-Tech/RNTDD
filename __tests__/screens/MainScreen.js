import MainScreen from '../../screens/MainScreen'
import React from 'react';
import renderer from 'react-test-renderer';


describe('Main screen', () => {
    it(' takes snapshot', () => {
        const tree = renderer.create(<MainScreen/>).toJSON();
        expect(tree).toMatchSnapshot();
      });    
});
