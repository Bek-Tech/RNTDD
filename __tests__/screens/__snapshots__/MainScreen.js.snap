// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main screen  takes snapshot 1`] = `
<View>
  <Text>
    main screen
  </Text>
</View>
`;
